contracts/Market.sol

```diff
@@ -110,8 +118,8 @@ contract Market is IMarket, Ownable, ERC721 {
 		return _getExpiry(index);
 	}
 
-	function getMarketTotal(bytes16 marketId) external view returns (uint256) {
-		return _marketTotal[marketId];
+	function getMarketTotalWagers(bytes16 marketId) external view returns (uint256) {
```

Comment:
Not sure i like the name.  Maybe totalValueLocked?

Reply:
It represents the total amount of wager amounts that have been placed for the given market. It used to be `getMarketTotal` but I decided to make it more explicit. `totalValueLocked` sounds ambiguous to me. Why would you prefer this term?